### ISSUE TITLE
Textbox color RGB

### ISSUE THREAD

**Issue by buarrows** [silexlabs/Silex#460]
It doesn't seem possible to change the color with specific RGB - there are only set colors. Is there a setting to change this?

**Comment by lexoyo** [comment]
I believe you are talking about this number here?

![screenshot from 2016-08-11 09-57-14](https://cloud.githubusercontent.com/assets/715377/17582089/06f12bea-5faa-11e6-9d0f-ddd7a820b661.png)

The last 2 digits are the alpha / transparency, so you have to add `FF` after your RGB value

**Comment by buarrows** [comment]
Yes. This is for background color but I can't get this menu when I edit text. I only have a fixed swatch selection.



Sent from my iPhone

On Aug 11, 2016, at 3:59 AM, Alex Hoyau <notifications@github.com<mailto:notifications@github.com>> wrote:


I believe you are talking about this number here?

[screenshot from 2016-08-11 09-57-14]<https://cloud.githubusercontent.com/assets/715377/17582089/06f12bea-5faa-11e6-9d0f-ddd7a820b661.png>

The last 2 digits are the alpha / transparency, so you have to add FF after your RGB value

-

**Comment by lexoyo** [comment]
Ok I see
For text colors it is not as simple
You can use CSS to style the text with RGB colors. For example the links (`a { color: #123456; }`) or you can add a style in the text editor (H1, H2, H3, Normal) and style with `h1 {}`, see https://github.com/silexlabs/Silex/issues/184#user-content-properties-css-panel

**Comment by buarrows** [comment]
Thanks! I was able to find a fix – didn’t quite get the heading working but was able to get the color in.

I’ll ask later more details, but in the meantime there is another question.

How do I merge elements? For example, I have some text together with an image on a background box, but when I show mobile view, the text boxes jump to the bottom of the screen.

is there a way to call up the entire html code so I can edit and merge elements? We don’t want everything to jump to the bottom though after it is merged.

Thanks so much for your help!

Joyce

**Comment by lexoyo** [comment]
> How do I merge elements? For example, I have some text together with an image on a background box, but when I show mobile view, the text boxes jump to the bottom of the screen. is there a way to call up the entire html code so I can edit and merge elements? We don’t want everything to jump to the bottom though after it is merged. 

can you put the image on the background of the text box? or on the background of a container and the text box in the container?

**Comment by singchan** [comment]
is this problem solved?

**Comment by buarrows** [comment]
Yes I was able to find a workaround.

It didn’t exactly work the way you said, but for now it’s okay.

Another question is: is there a way to copy the html code of the page so it can be paste into another program?

Thanks so much

jyw

**Comment by lexoyo** [comment]
FYI we are about to start a webinar in french
https://twitter.com/silexlabs/status/765961740752457728

**Comment by buarrows** [comment]
I saw the webinar, but I was interested in how to copy the html of the silex code into mailchimp so it could be mailed out as a webpage.

Also, what is the Normal font in Silex? Is it Calibri?

Thanks so much,
Joyce

From: Joyce Wong <arrows@bu.edu>
Date: Wednesday, August 17, 2016 at 12:45 PM
To: silexlabs/Silex <reply@reply.github.com>, silexlabs/Silex <Silex@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: Re: [silexlabs/Silex] Textbox color RGB (#460)

Yes I was able to find a workaround.

It didn’t exactly work the way you said, but for now it’s okay.

Another question is: is there a way to copy the html code of the page so it can be paste into another program?

Thanks so much

jyw
